packages/app/src/specs/SpecsList.cy.tsx

```diff
@@ -121,6 +121,23 @@ describe('<SpecsList />', { keystrokeDelay: 0 }, () => {
       cy.get('@showCreateSpecModalSpy').should('have.been.calledOnce')
     })
 
+    it('displays as expected at various widths', () => {
```

Comment:
Two things about this, which I can correct myself when back in the office tomorrow or somebody is free to take over:

1. This test should be combined with the `when screen is narrow` type tests lower down in this file
1. The Percy snapshots probably need to weight for the git info to be visible so we can confirm the overflow behavior, [this snapshot here ](https://percy.io/cypress-io/cypress/builds/19277314/changed/1082226949?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=2000&widths=500%2C650%2C800%2C1000%2C1200%2C1280%2C1500%2C2000)is the right size but since the content isn't rendered it doesn't capture the overflow.

Reply:
Moved things around a little and added a wait to allow for our 50ms delay built into rendering some of the row contents.